### DISCUSSION TITLE
How to customize the cookie name after login

### DISCUSSION THREAD

**Discussion by gaojianzhuang** [OrchardCMS/OrchardCore#16690] [Q&A]
When we log in the site, the client side will record the cookie as below:
![image](https://github.com/user-attachments/assets/3faea08d-f771-4169-bc68-c9f6a9192f8e)

How to define the cookie name instead of the "orchauth_Default"?
When I launch 2 orchard sites on my local, the cookie names will conflict with each other.

**Comment by hishamco** [comment]
You can configure cookie options

```csharp
services.ConfigureApplicationCookie(options =>
{
    options.Cookie.Name = "orchauth_" + HttpUtility.UrlEncode(_tenantName);
});
```
